modules/components/pages/inputs/aws_s3.adoc

```diff
@@ -92,7 +92,7 @@ When using SQS please make sure you have sensible values for `sqs.max_messages`
 
 == Download large files
 
-When downloading large files it's often necessary to process it in streamed parts in order to avoid loading the entire file in memory at a given time. In order to do this a <<codec, `codec`>> can be specified that determines how to break the input into smaller individual messages.
+When downloading large files it's often necessary to process it in streamed parts in order to avoid loading the entire file in memory at a given time. In order to do this a <<scanner, `scanner`>> can be specified that determines how to break the input into smaller individual messages.
```

Comment:
```suggestion
When downloading large files it's often necessary to process it in streamed parts in order to avoid loading the entire file in memory at a given time. To do this a <<scanner, `scanner`>> can be specified, which determines how to break the input into smaller individual messages.
```